### ISSUE TITLE
Test failure: SyntaxTests.testLiterals:188 ...

### ISSUE THREAD

**Issue by BurtHarris** [cincheo/jsweet#157]
After pulling down the latest changes, running the maven test target produced this error message:

```
Results :

Failed tests:
  SyntaxTests.testLiterals:188->AbstractTest.eval:178->AbstractTest.eval:183->AbstractTest.eval:203 exception occured while running test SyntaxTests.testLiterals with module kind none

Tests run: 182, Failures: 1, Errors: 0, Skipped: 11
```

I hope that's enough to be helpful.  Here's what looks like the error output file:

[org.jsweet.test.transpiler.SyntaxTests.txt](https://github.com/cincheo/jsweet/files/450910/org.jsweet.test.transpiler.SyntaxTests.txt)

**Comment by BurtHarris** [comment]
These messages look related:

```
ERROR: unclosed character literal at src\test\java\source\syntax\Literals.java(39)
2016-09-01 16:16:27.027 ERROR JSweetTranspiler:323 - src\test\java\source\syntax\Literals.java:39: error: unclosed character literal
		char c = 'Ã©';
		         ^
ERROR: illegal character: '\u00a9' at src\test\java\source\syntax\Literals.java(39)
2016-09-01 16:16:27.027 ERROR JSweetTranspiler:323 - src\test\java\source\syntax\Literals.java:39: error: illegal character: '\u00a9'
		char c = 'Ã©';
		           ^
ERROR: unclosed character literal at src\test\java\source\syntax\Literals.java(39)
2016-09-01 16:16:27.027 ERROR JSweetTranspiler:323 - src\test\java\source\syntax\Literals.java:39: error: unclosed character literal
		char c = 'Ã©';
		            ^
```
Note: I'm building on Windows 10.

**Comment by renaudpawlak** [comment]
We have had encoding issues under Windows (see #103). It's weird that it happens after the recent merge but I don't think it is related. Was it working for you before? My advice would be to skip the test until we solve the issue (of course if you can solve it it would be even better ;)). Thanks for reporting.

**Comment by lgrignon** [comment]
Fixed for 1.2.1 through df2286be5d7ca29dbb968b252ae06317137ea443 and deployed in Maven
